pkg/arvo/sys/vane/ames.hoon

```diff
@@ -4449,7 +4453,7 @@
             ::  if this was a re-send, don't adjust rtt or downstream state
             ::
             ?:  (gth tries.packet-state 1)
-              metrics
+              metrics(rto (clamp-rto (add rtt (mul 4 rttvar))))
```

Comment:
What is this doing?

Reply:
This is a merge issue I between develop and 413 branch I did not touch this